src/atoms/Checkbox/Checkbox.native.js

```diff
@@ -0,0 +1,238 @@
+  const theme = useContext(ThemeContext);
+
+  const onPressIn = () => {
+    let colorKey = 'tone.940';
+    if (isChecked) {
+      colorKey = `${variantColor || 'primary'}.930`;
+    }
+    const newUnderlayColor = getColor(theme, colorKey);
+    setUnderlayColor(newUnderlayColor);
+  };
+
+  const onPressOut = () => {
+    setUnderlayColor('transparent');
+  };
+
+  const onPress = () => {
+    if (isPropDefined(checked)) {
+      onChange(!isChecked);
+      return;
+    }
+    setCheckboxState((prevCheckboxState) => {
+      onChange(!prevCheckboxState);
+      return !prevCheckboxState;
+    });
+  };
+
+  if (disabled) {
+    titleTextColor = 'shade.950';
+    helpTextColor = 'shade.930';
+  }
+
+  return (
+    <Flex alignSelf="flex-start">
+      <TouchableOpacity
+        activeOpacity={1}
+        accessibilityRole="checkbox"
+        onPress={onPress}
+        underlayColor="transparent"
+        disabled={disabled}
+        onPressIn={onPressIn}
+        onPressOut={onPressOut}
+        {...automation(testID)}
+      >
+        <Flex flexDirection="row" alignItems="center">
+          <View>
+            <Backdrop
+              width={styles.backdrop.dimensions(size).width}
+              height={styles.backdrop.dimensions(size).height}
+              borderRadius={styles.backdrop.dimensions(size).borderRadius}
+              backgroundColor={underlayColor}
+            >
+              <Icon
+                size={size}
+                name={isChecked ? 'checkboxFilled' : 'checkboxOutlined'}
+                fill={styles.icon.fill({ isChecked, disabled, variantColor })}
+              />
+            </Backdrop>
+            <Flex alignSelf="center">
+              <Space margin={styles.title.margin()}>
+                <View>
+                  <Text color={titleTextColor} size={size}>
+                    {title}
+                  </Text>
+                </View>
+              </Space>
+            </Flex>
+          </View>
+        </Flex>
+
+        {(!isEmpty(helpText) || !isEmpty(errorText)) && size !== 'small' ? (
+          <Space margin={styles.helpText.margin(size)}>
+            <View>
+              <Text
+                size={styles.helpText.size(size)}
+                color={errorText ? 'negative.900' : helpTextColor}
+              >
+                {errorText || helpText}
+              </Text>
+            </View>
+          </Space>
+        ) : null}
+      </TouchableOpacity>
+    </Flex>
+  );
+};
+
+Checkbox.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  checked: PropTypes.bool,
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  title: PropTypes.string.isRequired,
+  helpText: PropTypes.string,
+  disabled: PropTypes.bool,
+  variantColor: PropTypes.oneOf(getVariantColorKeys()),
+  errorText: PropTypes.string,
+  testID: PropTypes.string,
+  defaultChecked: PropTypes.bool,
+};
+
+Checkbox.defaultProps = {
+  defaultChecked: undefined,
+  checked: undefined,
```

Comment:
Isn't this the same as not defining them here?

Reply:
All props which are not marked as `isRequired` should have default values.